README.md

```diff
@@ -118,7 +120,7 @@ $ sudo yum -y install trivy
 or
 
 ```
-$ rpm -ivh https://github.com/aquasecurity/trivy/releases/download/v0.1.6/trivy_0.1.6_Linux-64bit.rpm
+$ rpm -ivh https://github.com/aquasecurity/trivy/releases/download/{TRIVY_VERSION}/trivy_0.1.6_Linux-64bit.rpm
```

Comment:
there's still 0.1.6 in the file name that would need to be replaced as well.

Reply:
Thanks @mrueg for catching that!